content/md/en/docs/tutorials/get-started/upgrade-a-running-network.md

```diff
@@ -97,23 +116,31 @@ To start the node with the current runtime:
    Leave this node running.
    You can edit and re-compile to upgrade the runtime without stopping or restarting the running node.
 
-2. Open the [Polkadot/Substrate Portal](https://polkadot.js.org/apps/#/explorer) in a browser and connect to the local node.
+1. Open the [Polkadot/Substrate Portal](https://polkadot.js.org/apps/#/explorer) in a browser and connect to the local node.
+
+   To do this, click on the left-most dropdown menu to select the network.
+
+   ![Select network](/media/images/docs/tutorials/forkless-upgrade/polkadot-js-select-network.png)
+
+   Then scroll to the bottom of the list, click on `Development`, and select `Local Node`, then press the `Switch` button.
```

Comment:
```suggestion
1. Under **Development**, select **Local Node**, then click **Switch**.
```

Typically, I use bold for UI elements instead of back ticks.